pkg/istio/rbac/common/servicerolebinding.go

```diff
@@ -18,16 +18,18 @@ func GetServiceRoleBindingSpec(athenzDomainName string, roleName string, k8sRole
 
 		//TODO: handle member.Expiration for expired members, for now ignore expiration
 
-		spiffeName, err := MemberToSpiffe(member)
+		spiffeNames, err := MemberToSpiffe(member, false)
 		if err != nil {
 			log.Warningln(err.Error())
 			continue
 		}
 
-		spiffeSubject := &v1alpha1.Subject{
-			User: spiffeName,
+		for _, name := range spiffeNames {
+			spiffeSubject := &v1alpha1.Subject{
+				User: name,
+			}
+			subjects = append(subjects, spiffeSubject)
 		}
```

Comment:
Can we use singular noun spiffeName instead of name here?
```
for _, spiffeName := range spiffeNames {
        spiffeSubject := &v1alpha1.Subject{
            User: spiffeName,
        }
        subjects = append(subjects, spiffeSubject)
}
```